CD release notes 800xx  CDS-74699 (#2536)
harness/developer-hub: release-notes/continuous-delivery.md

  <TabItem value="What's new">
```

This release does not include new features. 

```mdx-code-block
  </TabItem>
  <TabItem value="Early access">
```

This release does not include early access features.

  
```mdx-code-block
  </TabItem>
  <TabItem value="Fixed issues">
```

> - Fixed a UI issue causing the stage dropdown options in the **Tests** tab of the execution page to scroll unexpectedly when an execution is in progress. (CDS-74026, ZD-47103)	

Reviewer: ```suggestion
- Fixed a UI issue causing the **Test Executions** stage dropdown options in the Execution page to scroll unexpectedly when an execution is in progress. (CDS-74026, ZD-47103)	
```

> - Fixed a UI issue in the **File Store** page where clicking on an entity link redirected to the Services page. With this fix, an entity link now points to the details page for the referenced entity. (CDS-73834, ZD-46193)	

Reviewer: ```suggestion
- Fixed a UI issue in the **File Store** page where clicking on an entity link redirected to the **Services** page. With this fix, an entity link now points to the **Details** page for the referenced entity. (CDS-73834, ZD-46193)	
```

> - Fixed a UI issue to ensure that the pipeline execution UI shows correct icons for container steps. (CDS-73725, ZD-47103)	

Reviewer: ```suggestion
- Fixed a UI issue to ensure that the pipeline execution UI shows correct icons for Container steps. (CDS-73725, ZD-47103)	
```

> - Harness does not currently support using expressions in failure strategies, so this support has been removed from the UI. Harness has a roadmap item to simplify YAML definitionss, which will support using expressions in failure strategies. (CDS-73614)	

Reviewer: ```suggestion
- Harness does not currently support using expressions in failure strategies, so this support has been removed from the UI. Harness has a roadmap item to simplify YAML definitions, which will support using expressions in failure strategies. (CDS-73614)	
```

- Fixed an issue with the `/ng/api/environmentsV2` endpoint. Previously, the endpoint would ignore overrides in YAML payloads when posting a request to create an environment. This endpoint now supports overrides in YAML environment definitions, as shown in the following example. (CDS-73496)

  ```yaml
  environment:
    name: coola
    identifier: coola
    tags: {}
    type: Production
    orgIdentifier: default
    projectIdentifier: H
    overrides:
      manifests:
      - manifest:
          identifier: sdda
          type: Values
          spec:
            store:
              type: Harness
              spec:
                files:
                - account:/s	
  ```

> - Fixed an issue where using selective stage execution in the advanced settings of a pipeline would cause the pipeline build to fail. This was due to incorrect index handling when processing `<+pipeline>` variables in shell scripts, which would result in index-array-out-of-bounds errors. (CDS-72840)	

Reviewer: Fixed an issue where the selective stage execution in a pipeline caused the pipeline build to fail. This was due to incorrect index handling when processing `<+pipeline>` variables in shell scripts, which would result in index-array-out-of-bounds errors. (CDS-72840)	
```

> - Fixed an API issue where a request to update a remote pipeline/InputSets did not update the `lastUpdateAt` field in the pipeline/InputSets. (CDS-72098)	

Reviewer: ```suggestion
- Fixed an API issue where a request to update the input sets of pipeline when importing the pipeline from Git did not update the `lastUpdateAt` field. (CDS-72098)	
```

> - Fixed an issue where Jira and ServiceNow approvals didn't fail fast if a connector provided was an incorrect type or not present. The pipeline would repeatedly request details of the Jira/ServiceNow ticket and keep failing with the same error (connector not found or incorrect connector).

Reviewer: ```suggestion
- Jira and ServiceNow approvals didn't fail fast if the connector provided was an incorrect type or not present. (CDS-69683)

The pipeline repeatedly requested details of the Jira/ServiceNow ticket and kept failing with the same error (connector not found or incorrect connector). This issue is fixed.
```

>   With this fix, the pipeline fails at the very beginning of the step execution if the connector type is incorrect or not present. This avoids the repeated polling and the delayed failure. (CDS-69683)

Reviewer: ```suggestion
  With this fix, the pipeline fails at the very beginning of the step execution if the connector type is incorrect or not present. This avoids the repeated polling and the delayed failure.
```
